blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
+
+Enabling the JMX API
+--------------------
+
+.. highlight:: none
+
+To monitor CrateDB using the JMX API, you must to set the following system properties before you start CrateDB::
```

Comment:
it was just a minor comment or idea. I meant links to the sections of this particular doc